src/client/components/email-form/email-form.vue

```diff
@@ -13,9 +13,11 @@
     <rich-text :id="messageId" :text="form.confirmationMessage" class="alert alert--success email-form__alert" />
     <rich-text v-if="form.body" :text="form.body" class="email-form__body" />
 
-    <label v-if="form.labelEmail" class="email-form__label-email">{{ form.labelEmail }}</label>
-    <input class="email-form__email form-input" type="email" name="email" placeholder="email@example.com" required>
-    <button class="email-form__submit button button--primary" type="submit">{{ form.labelSubmitButton }}</button>
+    <div class="email-form__field form-field">
```

Comment:
`email-form__field` is unused.

Reply:
Ah, missed that. Removed the unused class.
Also added a margin bottom to the email input on mobile to add spacing between the email input and button.